core/src/main/java/org/dcsa/conformance/core/party/ConformanceParty.java

```diff
@@ -97,7 +100,9 @@ public void importJsonState(JsonNode jsonState) {
     importPartyJsonState((ObjectNode) jsonState);
   }
 
-  protected abstract void importPartyJsonState(ObjectNode sourceObjectNode);
+  protected void importPartyJsonState(ObjectNode sourceObjectNode) {
+    // By default, nothing to import. Overwrite if needed
+  }
```

Comment:
I found 19 occurrences where this method is overwritten. About 8 times it is implemented. If it is not abstract, but has a default empty implementation, we can reduce the duplicated code and empty implementations quite a bit. 

So, @nt-gt and @gj0dcsa do you agree with this change also on the other method `exportPartyJsonState`?

Reply:
Yes, I understand. I was in doubt on how important that is here, especially because I don't fully functionally know where it is needed to implement and where not. Anyways, that's beyond the scope of this chat. 

I'll revert these 2 methods than, and accept that we have some duplicated code, e.g. in my Adoption API PR, failing the build.